pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -32,18 +31,16 @@
 import h5py
 from scipy import stats
 from pycbc.detector import Detector
-import pycbc.workflow as _workflow
-from pycbc.workflow.core import resolve_url_to_file
 # All/most of these final imports will become obsolete with hdf5 switch
 try:
     from ligo import segments
     from ligo.lw import utils, lsctables
     from ligo.lw.table import Table
     from ligo.segments.utils import fromsegwizard
-    # Handle MultiInspiral xml-talbes with glue,
+    # Handle MultiInspiral xml-tables with glue,
     # as ligo.lw no longer supports them
     from glue.ligolw import lsctables as glsctables
```

Comment:
Is `glsctables` used anywhere still?

Reply:
There are some post-processing codes we still need to revamp that in principle do use `glsctables` via `pycbc/results/pygrb_postprocessing_utils.py`.  See: https://github.com/gwastro/pycbc/issues/4419.

This is one of those cases where these scripts have been pulled out of the workflow generator, but you can run them from command line on old xml results file.  However, if we were to remove `glsctables` here, for example, automated tests on those scripts would fail; if we were to remove those scripts entirely, then 80% of their lines would be readded later.  We are in the middle of a transition.  I'd rather keep them, update them according to the issue I linked, and progressively clean up `pycbc/results/pygrb_postprocessing_utils.py`.  This is in line with what was done with `pycbc/results/grb_utils.py` for example.

I hope that makes sense.